RTNeural/common.h

```diff
@@ -42,6 +42,13 @@ static inline T tanh_approx(T x) noexcept
     auto denominator = (T)2027025 + x2 * ((T)945945 + x2 * ((T)51975 + x2 * ((T)630 + x2)));
     return numerator / denominator;
 }
+
+/** Hendrycks et al. approximation of GELU activation function. */
+template <typename T>
+static inline T gelu_approx(T x) noexcept
+{
+    return 0.5 * x * (1 + std::tanh((std::sqrt(2 / std::acos(-1))) * (x + (0.044715 * (std::pow(x, 3))))));
```

Comment:
A couple things:
- Would it be possible to link to the paper where the approximation comes from?
- `std::sqrt(2 / std::acos(-1))` is constant so we should pre-compute that value.
- `std::pow(x, 3)` would probably be faster as `x * x * x` (the benchmarking tool could confirm this)
- From the CI failures, it looks like some include files might be missing?